src/OpenTelemetry.Extensions/OpenTelemetry.Extensions.csproj

```diff
@@ -21,6 +21,8 @@
 
   <ItemGroup>
     <PackageReference Include="OpenTelemetry" Version="$(OpenTelemetryCoreLatestVersion)" />
+    <PackageReference Include="OpenTelemetry.Exporter.OpenTelemetryProtocol" Version="$(OpenTelemetryCoreLatestVersion)" />
+    <PackageReference Include="System.Text.Json" Version="9.0.4" />
```

Comment:
There are specific properties to use to determine which version to use:

https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/5298d31b00b40fe32ee9fe9f12e813cebe52ccdb/build/Common.props#L65-L69

Example usage:

https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/5298d31b00b40fe32ee9fe9f12e813cebe52ccdb/src/OpenTelemetry.Exporter.InfluxDB/OpenTelemetry.Exporter.InfluxDB.csproj#L21-L22

This package shouldn't force all TFMs to use the latest version as it can cause issues (e.g. https://github.com/open-telemetry/opentelemetry-dotnet/issues/5973).